0074

```diff
@@ -0,0 +1,164 @@
+<li>Divide the total number of MOBILE emitted during the Rewards Period by the sum of multiples of *estimated_reward_point* and *speedtest_multiplier* for each Radios to determine reward per one estimated_reward_point for Radio with Acceptable Speed Test.</li>
+<li>Sum up all the points for each Radio and multiply that number by the MOBILE per one *estimated_reward_point* to determine the reward for each Radio.</li>
+</ol>
+
+The new formula for Reward calculation per Radio:
+
+*heartbeat_multplier * speedtest_multiplier * (estimated_coverage_points * reward_per_one_estimated_coverage_point)*
+
+### Calculation Example
+
+For simplicity, assuming that total MOBILE Rewards per period is 10,000, and Radios provide coverage in 4 hexes at most. The actual projected coverage on average will include much more hexes.
+
+| Heartbeat | Speed Test | Estimated Coverage Reward Points
+---|---|---|---
+Radio 1 (Outdoor) | Acceptable (1x) | Hex1: 16, Hex2: 8. Total: 34
+Radio 2 (Outdoor) | Poor (0.25x) | Hex2: 8, Hex3: 4. Total: 12
+Radio 3 (Indoor) | Degraded (0.5x) | Hex1: 1, Hex2: 1, Hex 3: 1, Hex4: 4. Total: 7
+
+Table 3. Simplified data for two Outdoor and one Indoor Radio with Heartbeat, Speed Test and Estimated Coverage Points for one Reward Period.
+
+
+*R x (1x1x34 + 1x0.25x12 + 1x0.5x7) = 10,000,*
+
+Where R is the reward per one estimated coverage reward point.
+
+*R = 246.91*
+
+| Total MOBILE Rewards
+---|---
+Radio 1 (Outdoor) | = 246.91x1x1x34 = 8,395
+Radio 2 (Outdoor) | = 246.91x1x0.25x12 = 741
+Radio 3 (Indoor) | = 246.91x1x0.5x7 = 864
+
+
+## Data Visualization
+
+All members of the MOBILE SubDAO need to see real-life coverage provided by 5G Radios. Modeled Coverage data will play a key role in providing information to visualize it. More data will be added as additional external sources like Mobile Mappers come into play.
+
+This HIP proposes the creation of a new Explorer dedicated exclusively to MOBILE Network. As we grow with more features and data for rewards, combining it with information about the IoT Network becomes practically impossible.
+
+A basic map overlay of 5G data coverage with signal strength is proposed to be the first iteration of the Mobile SubDAO Explorer. Below is the visualization of signal propagation for the Outdoor Radio, directed along the street and installed on a pole on the two-story building roof.
+
+![Image 2. Baicells Outdoor Radio 430 installed on the two-story building roof on the pole.](<>)
+Image 2. Baicells Outdoor Radio 430 installed on the two-story building roof on the pole.
+
+![Image 3. Modeled coverage of the Radio pictured on Image 1.](<>)
+Image 3. Modeled coverage of the Radio pictured on Image 1.
+
+
+## Transition Plan
+
+To make sure the transition to the new PoC system, which replaces static, power-based multipliers, is smooth, Hotspot owners should be able to evaluate the coverage they provide before the switch takes effect.
```

Comment:
How can someone evaluate this? Can you elaborate this here?

Reply:
The plan is to allow users to locate their Radios in the new Mobile Explorer and see the modeled coverage from their Radio as an overlay on the map. The will be able to see how many hexed they cover, what are other Radios in that hex, and using the reference table, estimate the projected signal level and corresponding rewards.